src/util/physics.jl

```diff
@@ -82,21 +82,27 @@ end
 """ calc_line_width(λ, flux; frac_depth )
 Returns the line width (units of λ) for specified fractional line depth (default of 0.5).
 Assumes continuum is the maximum flux provided.
+Returns NaN if target flux cannot be found on either side of the line
 """
 function calc_line_width(λ::AbstractArray{T1,1}, flux::AbstractArray{T2,1}; frac_depth::Real = 0.5 ) where { T1<:Real, T2<:Real }
    @assert length(λ) == length(flux)
    @assert 0.05 <= frac_depth <= 0.99
-   idx_min_flux = argmin(flux)
+   #find the minimum flux, assuming it lies within the middle half of the flux array. This assumption makes it harder for nearby lines to contaminate this minimum.
+   idx_min_flux = argmin(flux[ Int(floor(length(flux) / 4)) : Int(ceil(length(flux) * 3 / 4)) ]) + Int(floor(length(flux) / 4)) - 1
```

Comment:
Please change code like `Int(floor(length(flux) / 4))` to 
`floor(Int,length(flux)//4))`.